Tools/scripts/parse_html5_entities.py

```diff
@@ -1,114 +0,0 @@
-#!/usr/bin/env python3
-"""
-Utility for parsing HTML5 entity definitions available from:
-
-    https://html.spec.whatwg.org/entities.json
-    https://html.spec.whatwg.org/multipage/named-characters.html
-
-The page now contains the following note:
-
-    "This list is static and will not be expanded or changed in the future."
```

Comment:
This script generates the list of HTML5 entities in `Lib/html/entities.py`.  In that file there are some comments there that reference this and that should be updated if this script is removed.

Initially the script was supposed to be run every once in a while to update the list, but the specs got recently updated with this note, so the only reason for keeping this script is to show how the list was generated (and possibly update it in the unlikely case that the upstream list changes).

FWIW I revised the script recently and removed the previous script (`Tools/scripts/parseentities.py`) in this PR:
* #92504